snippets/card-product.liquid

```diff
@@ -4,6 +4,7 @@
   Accepts:
   - card_product: {Object} Product Liquid object (optional)
   - media_aspect_ratio: {String} Size of the product image card. Values are "square" and "portrait". Default is "square" (optional)
+  - media_shape: {String} Image mask to apply to the product image card. Values are "blob", "chevronleft", "chevronright", "diamond", "parallelogram", and "round". (optional)
```

Comment:
This should've been `image_shape`, right?
```suggestion
  - image_shape: {String} Image mask to apply to the product image card. Values are "arch", "blob", "chevronleft", "chevronright", "diamond", "parallelogram", and "round". (optional)
```

I also think we're missing the `arch` value in this one :D